gymnasium/wrappers/jax_to_torch.py

```diff
@@ -24,6 +24,19 @@
     import jax
     import jax.numpy as jnp
     from jax import dlpack as jax_dlpack
+
+    def _at_least_version(version: str, target: str) -> bool:
```

Comment:
could we use https://packaging.pypa.io/en/latest/version.html#packaging.version.Version

Reply:
I think we should first discuss the implications of #1298 before adding another dependency. Since parts of the library are broken with older pytorch versions anyways, simply raising the version requirements and getting rid of the flag seems viable again.